### DISCUSSION TITLE
Don't use Netlify / sanity to host photo-grid

### DISCUSSION THREAD

**Discussion by nicosomb** [kwickramasekara/photo-grid#77] [Q&A]
Thank you for photo-grid, which looks nice. 

Is that possible to not use sanity and netlify to host it? 

I tried to use it locally, but I need a Sanity account, even on localhost...

**Comment by kwickramasekara** [comment]
Hi @nicosomb thank you for the interest in Photo Grid. There is no hard requirement of Netlify or any hosting solution for the front-end site. However, there is a hard requirement on Sanity which is the headless-CMS that handles photo assets. This is required to enable a lot of the features Photo Grid offers such as the [backend dashboard](https://github.com/kwickramasekara/photo-grid/wiki/Features#feature-packed-dashboard), image resizing, CDN, metadata extraction among many other things. 

Sanity has a generous [free tier](https://www.sanity.io/pricing) which offers 100GB of asset hosting. The front-end site can also be hosted totally for free at places like Netlify and Vercel with their free tiers. In theory, you can host a Photo Grid site completely for free if it's a hobby project with normal traffic. 

I hope that answers your questions and we currently don't have any plans to move away from Sanity due to feature dependencies listed above.

For future, please use our [discussions](https://github.com/kwickramasekara/photo-grid/discussions) page for questions or general discussions as we want to keep the Issues section focused on features and bugs.
